### PR TITLE
Style copy button

### PR DESCRIPTION
**Pull request by MasterIceZ**
OK Anos merge me
https://github.com/phakphum-dev/otog-frontend/issues/71

### CODE REVIEW

**Comment by Anon-136** [comment]
doesn't help #71 :(

File: src/pages/problem/[id].tsx [hunk 1]
```diff
@@ -300,7 +300,6 @@ const ExampleRow = ({ input, output, row = 0 }: ExampleRowType) => {
           onClick={onInputCopy}
           icon={<CopyIcon />}
           size="sm"
-          variant="ghost"
```
Comment (Anon-136) [inline src/pages/problem/[id].tsx:303]: can you add padding so that it help #71

**Review by Anon-136 — CHANGES_REQUESTED** [review]

File: src/pages/problem/[id].tsx (hunk 1 above)
Comment (MasterIceZ) [reply src/pages/problem/[id].tsx:303]: No

**Comment by Anon-136** [comment]
fixed in 7c60727

**Pull request closed without merging**